UT4MasterServer.Web/src/pages/Admin/Clients/Clients.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <CrudPage title="Clients">
+        <template #add="p">
+            <AddClient :all-clients="clients" @cancel="p.cancel" @added="loadClients(); p.cancel();" />
+        </template>
+        <template #filters>
+            <input type="text" class="form-control" placeholder="Search..." v-model="filterText" />
+        </template>
+        <LoadingPanel :status="status" @load="loadClients" auto-load>
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>ID</th>
+                        <th />
+                    </tr>
+                </thead>
+                <tbody>
+                    <template v-for="client in filteredClients.slice(pageStart, pageEnd)" :key="objectHash(client)">
+                        <tr :class="{ 'table-light': client.editing }">
+                            <td>{{ client.name }}</td>
+                            <td>{{ client.id }}</td>

```

Comment:
I would swap these two
```suggestion
                            <td>{{ client.id }}</td>
                            <td>{{ client.name }}</td>
```